src/02-common-principles.md

```diff
@@ -152,6 +152,15 @@ correspond to a unique identifier of that subject, such as `01`.
 The same holds for the `session` entity with its `ses-` key and its `<label>`
 value.
 
+The extra session layer (at least one `/ses-<label>` subfolder) SHOULD
```

Comment:
I have removed the stuff about labels.

Now there is only this sentence about folders in a section that is about filename but I think I can live with that.